Formula/vtk.rb

```diff
@@ -80,6 +87,9 @@ def install
       -DVTK_GROUP_ENABLE_Qt:STRING=YES
     ]
 
+    # https://github.com/Homebrew/linuxbrew-core/pull/21654#issuecomment-738549701
+    args << "-DOpenGL_GL_PREFERENCE=LEGACY"
```

Comment:
Does this change anything about linking behaviour on macOS?

Reply:
I checked; no difference in `brew linkage` on macOS. This tells just vtk to use mesa on Linux, instead of linking against `libglvnd`, which is the "new way" of using mesa, as you can switch between multiple GL backends. As our whole stack still uses mesa, we will stay with a direct linkage to mesa (but might transition to `libglvnd` later).